uber/config.py

```diff
@@ -111,6 +111,15 @@ def get_attendee_price(self, dt):
     def get_group_price(self, dt):
         return self.get_attendee_price(dt) - self.GROUP_DISCOUNT
 
+    def get_badge_count_by_type(self, badge_type):
```

Comment:
In order to limit each day's badge type to a cap, I had to implement the ability to find things like `FRIDAY_BADGE_COUNT` on-the-fly, so we could automatically remove options as they sold out. This is what this function does, and it's called by the ever-expanding __getattr__ override, so all you need to do is set `friday_badge_stock` in your configuration to limit the number of Friday badges you sell.